tests/unit/amo/sagas/test_collections.js

```diff
@@ -487,4 +488,122 @@ describe(__filename, () => {
       ).not.toContain(closeFormOverlay(formOverlayId).type);
     });
   });
+
+  describe('createCollection', () => {
```

Comment:
Please see my comments above about why this is still two big test cases for `create` and `update` rather than one for `modify`.